templates/service.yaml

```diff
@@ -24,7 +24,16 @@ spec:
   externalTrafficPolicy: "{{ .Values.service.externalTrafficPolicy }}"
   {{- end }}
   ports:
-    {{- toYaml .Values.service.ports | nindent 4 }}
+    {{- range .Values.service.ports }}
+    - name: {{ .name }}
+      {{- if .port }}
+      port: {{ int .port }}
+      {{- end }}
+      targetPort: {{ int .targetPort }}
```

Comment:
I believe this should be 

```
- name: {{ .name }}
  port: {{ int .port }}
  {{- if .targetPort }}
  targetPort: {{ int .targetPort }}
  {{- end }}
  {{- if .nodePort }}
  nodePort: {{ int .nodePort }}
  {{- end }}
```
Since `spec.ports.port` is required and `targetPort` and `nodePort` are optional

Reply:
Good catch. I'll fix it.